collector/stats_channel.go

```diff
@@ -59,6 +59,22 @@ func ChannelStats(namespace string) StatsCollector {
 			}, labels),
 		},
 		{
+			val: func(c *channel) float64 { return c.E2eLatency.Percentiles[0]["value"] },
```

Comment:
i'm not sure thats possible if you have stats on for nsq, and are scraping topic stats.